ursabot/docker.py

```diff
@@ -275,6 +275,8 @@ def conda(*packages, files=tuple()):
     'libboost-system-dev',
     'python',
     'python-pip',
+    'python3',
```

Comment:
We'll need a nice way to handle python versions uniformly.

Reply:
yeah, I didn't want to depend on the conda version.